jazz_deployments/components/validation/common.js

```diff
@@ -0,0 +1,152 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+    Common Validation functions for Environments-Catalog
+    @module: common.js
+    @description: Defines common validate functions.
+    @author:
+    @version: 1.0
+**/
+
+const _ = require("lodash");
+
+var validateIsEmptyInputData = function (deployment_data, onComplete) {
+    if (_.isEmpty(deployment_data)) {
```

Comment:
you can avoid using lodash here

Reply:
Using Object.keys(deployment_data).length === 0